src/main/webapp/content/theming/tumapplypreset.ts

```diff
@@ -683,5 +683,75 @@ export const TUMApplyPreset = definePreset(Lara, {
         },
       },
     },
+    tag: {
+      fontSize: '0.75rem',
+      alignItems: 'center',
+      gap: '0',
+      rounded: {
+        borderRadius: '9999rem',
```

Comment:
isn't this too large? Wouldn't 1rem have the same effect?
```suggestion
        borderRadius: '1rem',
```